mods/hv/rules/aircraft.yaml

```
@@ -811,17 +811,18 @@ BATTLESHIP:
 	Buildable:
 		Queue: Aircraft
 		BuildPaletteOrder: 100
-		Prerequisites: ~starport.yi, techcenter, ~disabled
+		BuildLimit: 1
+		Prerequisites: radar, trader, techcenter, ~starport.yi
 		Description: actor-battleship.description
 	UpdatesPlayerStatistics:
 		AddToArmyValue: true
 	Health:
-		HP: 100000
+		HP: 250000
 	Armor:
 		Type: Heavy
 	RevealsShroud:
-		Range: 12c0
-		MinRange: 4c0
+		Range: 16c0
+		MinRange: 6c0
```

```suggestion
		MinRange: 8c0
```

From what I've seen, `MinRange` is always the halved value of `Range`.

---

It's not always the case, but I think we should change to always be the half of `Range` for all units to be consistant.